ot_handler/init_pipette_for_tracking.py

```diff
@@ -0,0 +1,79 @@
+from types import MethodType
+from typing import Optional, Union
+from opentrons.types import Location
+from opentrons.protocol_api.instrument_context import InstrumentContext
+from .TrackedWell import TrackedWell
+
+def init_pipette_for_tracking(pipette):
+    """ Function to overwrite aspirate and dispense commands to enable volume tracking of tracked plates"""
+    
+    #Check if pipette is already initalized for volume tracking
+    if not hasattr(pipette, '_vol_tracking'):
+        pipette._vol_tracking = True
+        #pipette._aspirate = pipette.aspirate
+        #pipette._dispense = pipette.dispense
+
+        #Functions with volume tracking
+        def aspirate_tracked(self, volume: Optional[float] = None, location: Optional[Union[Location, TrackedWell]] = None, rate: float = 1.0, single_tip_mode=False) -> InstrumentContext:
+            """Aspirate liquid from a well, tracking volume if it's a TrackedWell"""
+            self.aspirate(volume, location, rate)
+            
+            if isinstance(location, TrackedWell):
+                if 'single' in pipette.name:
+                    location.parent.remove_volume(location.well_name, volume)
+                
+                elif single_tip_mode == True:
+                    location.parent.remove_volume(location.well_name, volume)
+                
+                elif "multi" in pipette.name and single_tip_mode == False:
+                    if location.parent.plate_data['plate_type'] == "reservoir":
```

Comment:
The nature of the well is currently checked by the well dimensions in the LiquidHandler. Basically, if the y dimension of the well exceeds a value, we can assume the multichannel pipette accesses with all the tips.